cmd/manager/main.go

```diff
@@ -105,18 +110,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	syncperiod := time.Duration(time.Minute)
```

Comment:
Yeah, I'm open to discussion on this time. If the grafanadashboard, servicemonitor and prometheusrule CRDs are created as part of an installation of application monitoring, how long should the user wait in order for the related crs to be created inside the reconcile?

there's also the argument that for application-monitoring it will already be installed by the time this operator is installed. We could not trigger it on a syncperiod and document it instead?

Reply:
> [...] how long should the user wait in order for the related crs to be created inside the reconcile?

Not trying to give a silly answer, but they should wait up to as long as whatever we set the sync period here :)

The reason I say that (as silly as it may sound), is that we shouldn't be setting this just based on these monitoring resources, since this will reconcile everything (all primary and secondary resources). Granted for this operator we've only got 1 CRD and only allow 1 CR for that, so in this case it won't be a huge amount of extra work if this is a short period. Think about the UPS operator though: it could have a lot of CRs to manage, which could cause a lot of extra unnecessary traffic to the api server if it's set too low.

I honestly don't know what a good period is here, I'm just being cautious. Maybe a minute is fine :) @davidffrench @maleck13 do you have any thoughts on it?